media-libs/openexr/files/openexr-2.5.7-0001-disable-testRgba-on-sparc.patch

```diff
@@ -0,0 +1,31 @@
+From: Bernd Waibel <waebbl-gentoo@posteo.net>
+Date: Mon, 21 Jun 2021 12:03:02 +0200
+Subject: [PATCH] disable testRgba on sparc
+
+Fails on sparc, bug #656680 and issue
+https://github.com/openexr/openexr/issues/222
+
+Bug: https://bugs.gentoo.org/656680
+Bug: https://github.com/AcademySoftwareFoundation/openexr/issues/222
+
+Signed-off-by: Bernd Waibel <waebbl-gentoo@posteo.net>
+---
+ IlmImfTest/main.cpp | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/IlmImfTest/main.cpp b/IlmImfTest/main.cpp
+index 3245fc4..3ba3e60 100644
+--- a/IlmImfTest/main.cpp
++++ b/IlmImfTest/main.cpp
+@@ -206,7 +206,7 @@ main (int argc, char *argv[])
+     TEST (testXdr, "core");
+     TEST (testHuf, "core");
+     TEST (testWav, "core");
+-    TEST (testRgba, "basic");
++//    TEST (testRgba, "basic");
```

Comment:
In future, just drop the line -- we lose no information because the diff still shows what was there in the past.

I'm a bit uneasy about disabling it because it could be a genuine issue in the code (i.e. the test is _exposing_ a problem), but I suppose it's only one. Let's keep an eye on the bug upstream.

Reply:
The upstream bug doesn't had any progress lately. I keep an eye on it. It probably is an issue dedicated only to sparc machines. In the upstream bug, someone said on other big endian architectures, the issue didn't show up.